lib/nimble_totp.ex

```diff
@@ -99,7 +99,7 @@ defmodule NimbleTOTP do
   def otpauth_uri(label, secret, uri_params \\ []) do
     key = Base.encode32(secret, padding: false)
     params = [{:secret, key} | uri_params]
-    query = URI.encode_query(params)
+    query = URI.encode_query(params, :rfc3986)
```

Comment:
Per https://hexdocs.pm/elixir/URI.html#encode_query/2, `:rfc3986` is only available since Elixir v1.12. We should either bump the Elixir requirement or work around it (with a reminder to remove it once we require Elixir v1.12)

@josevalim should we add workaround or bump requirement?

Reply:
ah, it was bumping requirement, good call.